app/templates/_Gruntfile.js

```diff
@@ -0,0 +1,106 @@
+module.exports = function(grunt) {
+
+  'use strict';
+
+  require('time-grunt')(grunt);
+
+  var path = require('path');
+  var extend = require('node.extend');
+
+
+  /**
+   * Load the tasks we want to use, which are specified as dependencies in
+   * the package.json file of cf-grunt-config.
+   */
+
+  // Loads all Grunt tasks in the node_modules directory within the new CWD.
+  require('jit-grunt')(grunt, {
+    // Below line needed because task name does not match package name
+    bower: 'grunt-bower-task'
+  })({
+    // Options
+    pluginsRoot: 'node_modules/cf-grunt-config/node_modules'
+  });
+
+
+  /**
+   * Initialize a variable to represent the Grunt task configuration.
+   */
+  var config = {
+
+    // Define a couple of utility variables that may be used in task options.
+    pkg: grunt.file.readJSON('bower.json'),
+    opt: {
+      // Include path to compiled extra CSS for IE7 and below.
+      // Definitely needed if this component depends on an icon font.
+      ltIE8Source: 'static/css/main.lt-ie8.min.css',
```

Comment:
Given that we're dropping support for IE7, how about we go ahead and comment this out?